### PR TITLE
Fix incorrect username

### PR DESCRIPTION
**Pull request by monopole**
For work on issue #598
My mistake - used goog username rather than github.  Duh.
Wish we had a federated identity authority we all trusted :P

```release-note
NONE
```

### CODE REVIEW

**Comment by monopole** [comment]
/assign @dchen1107 
/unassign @brendandburns 

See #47074

**Comment by mengqiy** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this

**Comment by mengqiy** [comment]
@monopole Need to change them in https://github.com/kubernetes/kubernetes/blob/master/build/visible_to/OWNERS as well.

**Comment by ixdy** [comment]
@monopole I think #598 may also be the wrong issue?

**Comment by mengqiy** [comment]
He means https://github.com/kubernetes/community/pull/598

File: OWNERS [hunk 1]
```diff
@@ -10,7 +10,7 @@ approvers:
   - brendandburns
   - dchen1107
   - jbeda
-  - jregan # To modify BUILD files per proposal #598
+  - monopole # To modify BUILD files per proposal #598
```
Comment (ixdy) [inline OWNERS]: update issue reference, too? kubernetes/community#598

**Comment by BenTheElder** [comment]
/retest

We should probably fix this still right? :-)

**Pull request closed without merging**